ccaas_builder/cmd/detect/main.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var logger = log.New(os.Stderr, "", 0)
+
+func main() {
+	logger.Println("::Detect")
+
+	if err := run(); err != nil {
+		logger.Printf("::Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	logger.Printf("::Type detected as ccaas")
+
+}
+
+type ChaincodeMetadata struct {
```

Comment:
Only use leading upper case if the type (or function) is exported (used outside of this package).

Reply:
Ah yes.. sorry forget there was a variant of Hungarian Notation instead of `public/private .. ` keywords